step.yml

```diff
@@ -198,6 +198,16 @@ inputs:
     - "no"
     is_required: true
 
+- manage_version_and_build_number: "no"
```

Comment:
Don't forget to run the `generate_readme` workflow to update the README with this new input!

Reply:
This PR #34 has made major changes to the readme.. should I just update that once this pr is merged?

Reply:
The generator gets data from step.yml, so if you add a new input in this PR, you should update the README too in the same PR as a rule of thumb